app/helpers/application_helper.rb

```diff
@@ -221,7 +221,9 @@ def additional_info(text)
 
   def page_title(content = nil, in_tab: false, &block)
     content ||= capture(&block)
-    content_for :page_title, strip_tags(content) + (@project ? " - #{@project.name}" : "")
+    title_content = strip_tags(content).gsub('&amp;', '&').html_safe
+    title_content << " - #{@project.name}" if @project
```

Comment:
Is it possible for the project name to have a character like `&` in it?

Reply:
stages can have that

Reply:
ah I see ... yes we should do that after :D

Reply:
fixed